schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
  date_of_birth DATE,
  escape_attempts INT,
  neutered BOOLEAN,
- weight_kg NUMERIC
+ weight_kg NUMERIC,
+ species TEXT
```

Comment:
- You seem to have manually edited your animal's table, I suggest you use the line ` ALTER TABLE animals ADD species varchar(255)` to dynamically update the table

Reply:
Fixed, thank you.